### PR TITLE
update api category packages 2

### PR DESCRIPTION
**Pull request by alharris-at**
#### Description of changes
Updating to latest published API Packages

#### Issue #, if available
N/A

#### Description of how you validated changes
Running `yarn test` and running e2e tests here https://app.circleci.com/pipelines/github/aws-amplify/amplify-cli?branch=run-e2e%2Falharris%2Fupdate-api-category-packages-2&filter=all

#### Checklist
- [x] PR description included
- [x] `yarn test` passes
- [x] Tests are [changed or added](https://github.com/aws-amplify/amplify-cli/blob/master/CONTRIBUTING.md#tests)
- [ ] Relevant documentation is changed or added (and PR referenced)
- [ ] New AWS SDK calls or CloudFormation actions have been added to relevant test and service IAM policies

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: packages/amplify-util-mock/src/__tests__/velocity/model-auth.test.ts [hunk 1]
```diff
@@ -777,82 +777,11 @@ describe("@model @primaryIndex @index auth", () => {
       updatedAt: AWSDateTime
     }`;
 
-    const out = transformer.transform(validSchema);
```
Comment (lazpavel) [inline packages/amplify-util-mock/src/__tests__/velocity/model-auth.test.ts:780]: is this change supposed to be part of the package update PR?

Comment (alharris-at) [reply packages/amplify-util-mock/src/__tests__/velocity/model-auth.test.ts:780]: Yeah, this was a test that Danielle had updated in the api category repo that needed to be reflected back here as well, so I just cherry-picked that file update in from her initial PR.

Comment (lazpavel) [reply packages/amplify-util-mock/src/__tests__/velocity/model-auth.test.ts:780]: thanks, LGTM, there is a conflict though

**Review by lazpavel — APPROVED** [review]

**Comment by danielleadams** [comment]
Fixing the conflict

**Review by sachscode — APPROVED** [review]

**Pull request merged**